src/Menu/menu_left.c

```diff
@@ -30,7 +32,14 @@ void menu_sub_8003B568(void)
     }
 }
 
-#pragma INCLUDE_ASM("asm/menu_rpk_8003B5E0.s") // 52 bytes
+
+
+MenuMan_Inventory_14h_Unk *menu_rpk_8003B5E0(int idx)
+{
+    return (MenuMan_Inventory_14h_Unk *)(gMenuMan_800BD360.field_154.code +
```

Comment:
I think this is actually `(MenuMan_Inventory_14h_Unk *)( &gMenuMan_800BD360.field_150 + gMenuItemRpkInfo_8009E484[ idx ].field_4_rpk_idx * 5 )`. This function is really confusing.

Reply:
This very nearly broke my brain.
It's accessing dword_800BD5A0. The lowest code in the gMenuItemRpkInfo_8009E484 indices us 12, so it's that index - 12.